src/Form.jsx

```diff
@@ -202,9 +202,9 @@ export default class Form extends React.Component {
         focused: false,
         disabled: prevDisabled
       }
-    }).then(() => {
+    }).then(({ nextProps }) => {
       /* Invoke custom onBlur handler */
-      if (onBlur) onBlur();
+      if (onBlur) onBlur(event, nextProps, this.props);
     });
   }
 
```

Comment:
Please, could you also add custom event handlers to the `DefaultForm.js` in the `examples/`?

Just a simple functions outputting the arguments into console:
```jsx
handleFieldChange = (...args) => console.log(args);

<MyInput onChange={ this.handleFieldChange } />
```
This way we have something like test. In the future there should be proper tests for doing so.